docs/sources/administration/configuration.md

```diff
@@ -772,6 +772,22 @@ Refer to [Auth proxy authentication]({{< relref "../auth/auth-proxy.md" >}}) for
 
 Refer to [LDAP authentication]({{< relref "../auth/ldap.md" >}}) for detailed instructions.
 
+## [aws]
+
+In this section, core and external AWS plugins can be configured.
```

Comment:
```suggestion
You can configure core and external AWS plugins.
```

Reply:
* Eliminate text that talks about the documentation meta layer (this section, this document, this paragraph, etc.); instead,  talk about the concepts and tasks directly.
* Replace passive voice with active voice and identify the agent (doer of the action) where possible.